PyStemmusScope/stemmus_scope.py

```diff
@@ -122,37 +124,47 @@ def setup(
         Args:
             WorkDir: path to a directory where input/output directories should be created.
             ForcingFileName: forcing file name. Forcing file should be in netcdf format.
-            NumberOfTimeSteps: total number of time steps in which model runs. It can be
-                `NA` or a number. Example `10` runs the model for 10 time steps.
+            StartTime: Start time of the model run. It must be in ISO format (e.g. 2007-01-01T00:00).
+            EndTime: End time of the model run. It must be in ISO format (e.g. 2007-01-01T00:00).
 
         Returns:
-            Paths to config file and input/output directories
+            Path to the config file
+            Forcing filename
         """
         # update config template if needed
         if WorkDir:
             self._config["WorkDir"] = WorkDir
 
-        if ForcingFileName:
-            self._config["ForcingFileName"] = ForcingFileName
+        if Location:
+            _ = utils.check_location_fmt(Location)
```

Comment:
let's move this check to `config_io.read_config()` function.

Reply:
I have also added the check there, but I think we need to keep the location format check in `StemmusScope.setup` as well, as the user can pass a new location input here, which should be checked for validity.
It does get checked again a few lines later, but this is after the creation of the new directories.

I've decided to put all the config checks into a wrapper "validate_config" function, so whenever we need to validate the config, or write new checks for the input, we can put the checks there. It might be good to also write a check that validates the directories in the config file.